packages/main/src/Popup.ts

```diff
@@ -188,6 +189,17 @@ abstract class Popup extends UI5Element {
 	@property({ defaultValue: "" })
 	accessibleNameRef!: string;
 
+	/**
+	 * Allows setting a custom role
```

Comment:
Update the comment to something like "Allows setting the role. Available options are:..." since this no longer allows to set an actual custom role.